lib/blockchain.go

```diff
@@ -2876,6 +2876,45 @@ func (bc *Blockchain) ValidateTransaction(
 	return nil
 }
 
+// ValidateTransactions creates a UtxoView and sees if the transactions can be connected
+// to it. If a mempool is provided, this function tries to find dependencies of the
+// passed-in transactions in the pool and connect them before trying to connect the
+// passed-in transactions.
+func (bc *Blockchain) ValidateTransactions(
+	txns []*MsgDeSoTxn, blockHeight uint32, verifySignatures bool, mempool Mempool) error {
+
+	// Create a new UtxoView. If we have access to a mempool object, use it to
+	// get an augmented view that factors in pending transactions.
+	utxoView, err := NewUtxoView(bc.db, bc.params, bc.postgres, bc.snapshot, bc.eventManager)
```

Comment:
Do we really want to pass in `bc.eventManager` here? Since this is a temp throwaway utxoView, we wouldn't want to trigger any events for downstream listeners
```suggestion
	utxoView, err := NewUtxoView(bc.db, bc.params, bc.postgres, bc.snapshot, nil)
```